pom.xml

```diff
@@ -99,6 +99,7 @@
     <google.http-client.version>1.39.2-sp.1</google.http-client.version>
     <google.oauth-client.version>1.31.5</google.oauth-client.version>
     <google.protobuf.version>3.17.3</google.protobuf.version>
+    <spotless.version>2.12.2</spotless.version>
```

Comment:
Usually we specify plugin versions in-line.

Reply:
Ack